src/containers/share-sound-to-tiktok-modal/ShareSoundToTikTokModal.tsx

```diff
@@ -41,9 +42,9 @@ const fileRequirementErrorMessages = {
 }
 
 const ShareSoundToTikTokModal = () => {
+  const [isOpen, setIsOpen] = useModalState('ShareSoundToTikTok')
```

Comment:
should this string be an exported const somewhere?

Reply:
`useModalState` accepts a union type so typescript checks that this is valid. This is how we are doing it in all the other instances

 It also autocompletes the string!

It might be valuable to discuss using union types in Stems too, because then you don't need an extra import to do `ButtonType.PRIMARY` for example. But it only makes sense if we expect all consumers of Stems to be using typescript